crosschain-resolver/contracts/L1Resolver.sol

```diff
@@ -27,9 +28,20 @@ contract L1Resolver is EVMFetchTarget, ITargetResolver, IExtendedResolver, ERC16
     uint256 constant VERSIONABLE_ADDRESSES_SLOT = 2;
     uint256 constant VERSIONABLE_HASHES_SLOT = 3;
     uint256 constant VERSIONABLE_TEXTS_SLOT = 10;
+    string  public   graphqlUrl;
+    string  public   resolverName;
+    uint256 public   l2ResolverCoinType;
 
-    event TargetSet(bytes32 indexed node, address target);
-
+    event TargetSet(bytes name, bytes32 indexed node, address target);
+    event MetadataChanged(
```

Comment:
Shouldn't this be part of the interface too?

Reply:
What's the general guidline of which functions to put in the interface and which is not? `TargetSet` event is currently not part of `ITargetResolver` interface though

Reply:
Generally, the interface is for consumption by integrators. So, public functions and events need to be in it; admin-only functions don't. We don't include set functions in resolver interfaces because the interface is defined in a spec that only defines read functionality, since different resolvers might have different ways to update records.